### PR TITLE
Fix typos in Docs

### PR DESCRIPTION
**Pull request by navidcy**
- `Docs/Physics/HydrostaticFreeSurfaceModel`
  At the moment docs mention that we obtain $w(x, y, z, t)$ by integrating the mass conservation equation from bottom to top but we need to integrate from bottom to depth $z$. This PR fixes this.

- `Docs/Physics/NonHydrostaticModel`
  The rotation rate term due to surface waves was missing a sign.

### CODE REVIEW

File: docs/src/physics/hydrostatic_free_surface_model.md [hunk 1]
```diff
@@ -26,7 +28,7 @@ condition at the surface
 
 ## The momentum conservation equation
 
-The equations governing the conservation of momentum in a rotating fluid, including buoyancy
+The equations governing the conservation of horizontal momentum in a rotating fluid, including buoyancy
```
Comment (simone-silvestri) [inline docs/src/physics/hydrostatic_free_surface_model.md:31]: These are conservation of the 3d momentum, no? (where conservation of vertical momentum is simplified by neglecting vertical acceleration as well as non-linear terms and results in the hydrostatic relation)

Comment (navidcy) [reply docs/src/physics/hydrostatic_free_surface_model.md:31]: ok, fair!

Comment (navidcy) [reply docs/src/physics/hydrostatic_free_surface_model.md]: [ff3af9b](https://github.com/CliMA/Oceananigans.jl/pull/3150/commits/ff3af9b6ba2582bc0a9ad634b3c097e7ed6daf26) resolved this

File: docs/src/physics/hydrostatic_free_surface_model.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ The mass conservation equation is
     0 = \boldsymbol{\nabla} \boldsymbol{\cdot} \boldsymbol{u} + \partial_z w \, .  
 ```
 
-The above is integrated from the bottom of the fluid up to ``z = 0`` to obtain ``w(x, y, z, t)``.
+Given the horizontal flow ``\boldsymbol{u}`` we use the above to diagnose the vertical velocity ``w``.
+We integrate the mass conservation equation from the bottom of the fluid (where ``w = 0``) up t
+depth ``z`` and recover ``w(x, y, z, t)``.
```
Comment (tomchor) [inline docs/src/physics/hydrostatic_free_surface_model.md:21]:
```suggestion
We integrate the mass conservation equation from the bottom of the fluid (where ``w = 0``) up to
depth ``z`` and recover ``w(x, y, z, t)``.
```

**Pull request merged**